kernel-boot/modules/rdma.conf

```diff
@@ -0,0 +1,21 @@
+# These modules are loaded by the system if any RDMA devices is installed
+# iSCSI over RDMA client support
+ib_iser
+
+# iSCSI over RDMA target support
+# ib_isert
+
+# User access to RDMA verbs (supports libibverbs)
+ib_uverbs
+
+# User access to RDMA connection management (supports librdmacm)
+rdma_ucm
+
+# RDS over RDMA support
+# rds_rdma
```

Comment:
Maybe this is  a better discussion for the mailing list..

This is a long path, this PR is just a part of the groundwork.

When we add in the patch I recently sent the mailing list that introduces rdma-hw.target, then we can talk about how to revise each of these services to autoload in some way.

For instance NFS client should be fixed with a kernel patch to autoload the modules when the nfs-rdma parameter is provided.

It looks like NFS server should be fixed by adding a drop in to nfs-server.service that triggers
rdma-load-modules@nfsd.service if rdma-hw.target has been pulled in, or maybe we need a nfs-server-rdma.service  that triggers it,  and does the other setup (??) I am not certain what is best. Consulting with Chuck would be a good idea.

No idea about RDS...

iser client and server should probably autoload at the kernel, from what I can tell..

But, at this point in the project, this series only mostly emulates the behaviour of existing rdma.conf by unconditionally auto-loading these modules. 

It does make made srp_daemon and iwpmd autoload their modules as a proof of concept.

I want to tackle this broader issues with other PRs, not in this one..